packages/cli/src/services/jwt.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Service } from 'typedi';
+import * as jwt from 'jsonwebtoken';
+
+@Service()
+export class JwtService {
```

Comment:
I'd suggest that we read the `jwtSecret` secret into a readonly property on this class, abstract away all JWT specific stuff into this class.